readme/apps/mfa.md

```diff
@@ -0,0 +1,59 @@
+# Multi-factor authentication
+
+You can now enable Multi-Factor Authentication (MFA) to your Joplin Account.
```

Comment:
For the whole document, please don't add line breaks in paragraphs. We generally don't rely on soft breaks and I don't know how scripts along the pipeline to build the website are going to handle them. So essentially each paragraph should be on a single line